search/processor.cpp

```diff
@@ -640,6 +640,18 @@ void Processor::Search(SearchParams params)
 
 bool Processor::SearchDebug()
 {
+  if (m_query.m_query == "?wiki")
+  {
+    EmitWithMetadata(feature::Metadata::FMD_WIKIPEDIA);
+    return true;
+  }
+
+  if (m_query.m_query == "?description")
+  {
+    EmitWithMetadata(feature::Metadata::FMD_DESCRIPTION);
+    return true;
+  }
```

Comment:
Put under DEBUG?

Reply:
The goal is to test it in the Release. And maybe play with some caching later.